src/ott/utils.py

```diff
@@ -59,3 +60,56 @@ def is_jax_array(obj: Any) -> bool:
     # https://jax.readthedocs.io/en/latest/jax_array_migration.html
     return isinstance(obj, (jax.Array, jnp.DeviceArray))
   return isinstance(obj, jnp.DeviceArray)
+
+
+def default_progress_fn(
+    status: Tuple[np.ndarray, np.ndarray, np.ndarray, NamedTuple], *args
+) -> None:
+  """This default callback function reports progress by printing to the console.
+
+  Args:
+      status: tuple describing the current iteration number,
```

Comment:
Status should be rendered as an rST list, similar to the description in `ProgressCallbackFn`.